packages/browser/src/builder/BacktraceClientBuilder.ts

```diff
@@ -13,7 +13,7 @@ import { SpiderMonkeyStackTraceConverter } from '../converters/SpiderMonkeyStack
 import { getEngine } from '../engineDetector';
 
 export class BacktraceClientBuilder extends BacktraceCoreClientBuilder<BacktraceClient> {
-    constructor(private readonly _options: BacktraceConfiguration) {
+    constructor(protected readonly _options: BacktraceConfiguration) {
```

Comment:
```suggestion
    constructor(protected readonly options: BacktraceConfiguration) {
```

1. if it is protected, can we remove `_` ?
2. Do you need it for any reason?

Reply:
1. Fixed
2. I do need it when creating a BacktraceClient in `BacktraceReactClientBuilder.build()`